### PR TITLE
Data attribute

### PR DESCRIPTION
**Pull request by pokorson**

### CODE REVIEW

File: problems/data_attribute/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html lang="en">
+  <head>
+      <meta charset="utf-8">
+      <title>tape</title>
+  </head>
+  <body>
+    <script>
+      var elements = document.getElementsByTagName('div');
```
Comment (maciekblim) [inline problems/data_attribute/index.html:9]: @pokorson why global `i` and `elements` ??

File: problems/data_attribute/problem.txt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+storing logic data. For such operation we should use HTML data attribute. Syntax looks like this
+
+{{ code }}
+  <p data-id="1">...</p>
+{{/ code }}
+
+We can use data attributes as selectors to find element.
+
+{{ code }}
+  $('[data-name="exampleName"]');
```
Comment (maciekblim) [inline problems/data_attribute/problem.txt]: @pokorson I don't know if teaching usage of that kind selectors is a good thing. IMO they are slow and generally we should rather avoid them (maybe some tip here?)

Comment (pokorson) [inline problems/data_attribute/problem.txt]: @blimusiek right, so should I change query to `class`/`id` and then just read `data-token`?

Comment (maciekblim) [inline problems/data_attribute/problem.txt]: @pokorson yeah I think using `class` and then reading `data-token` is better practice :)

**Comment by pokorson** [comment]
@blimusiek I've removed querying by `data-attribute` and added setting new attribute with jQuery

**Comment by maciekblim** [comment]
@pokorson cool ;)

**Pull request merged**